scripts/tests/calibnet_no_discovery_check.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+set -euxo pipefail
+
+# This script tests forest behaviours when discovery(mdns and kademlia) is disabled
+
+source "$(dirname "$0")/harness.sh"
+
+$FOREST_PATH --chain calibnet --encrypt-keystore false --mdns false --kademlia false --auto-download-snapshot --log-dir "$LOG_DIRECTORY" --detach --save-token ./admin_token
+FULLNODE_API_INFO="$(cat admin_token):/ip4/127.0.0.1/tcp/2345/http"
+export FULLNODE_API_INFO
+
+wait_util_rpc_is_ready
```

Comment:
`--detach` is supposed to wait until the RPC is ready. If that isn't happening then we should fix it.

Reply:
You are right, `--detach` does wait until the RPC port is listened on.